docs/installation.md

```diff
@@ -50,17 +50,30 @@ list of instruments:
         "platformType": "ILLUMINA",
         "name": "default",
         "timeZone": "America/Toronto",
-        "parameters": {}
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/run1"
+        ]
       },
       {
         "path": "/srv/sequencer/promethion",
         "platformType": "OXFORDNANOPORE",
         "name": "promethion",
         "timeZone": "America/Toronto",
-        "parameters": {}
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/test/run1",
+            "/run2"
+        ]
       }
     ]
 
+The ignoreSubdirectories field is used to exclude subdirectories from processing. For example, a 
+known bad run or a test folder holding temporary files like /test/mytestrun. This is optional 
```

Comment:
This `/test/mytestrun` isn't connected to anything and so isn't a meaningful example. Either reference the `/test/run1` in your example config, or just end at "or a test folder holding temporary files."